tdex/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: tdex
+category: Finance
+name: TDEX Provider
+version: "0.8.15"
+tagline: Provider liquidity on the TDEX Network, an open protocol to trade Liquid Network assets.
```

This should be provide, right?

```suggestion
tagline: Provide liquidity on the TDEX Network, an open protocol to trade Liquid Network assets.
```